io.qt.QtDesignStudio.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.qt.QtDesignStudio
+runtime: org.kde.Sdk
+runtime-version: '6.5'
+base: io.qt.qtwebengine.BaseApp
+base-version: '6.5'
+sdk: org.kde.Sdk
+command: qtdesignstudio
+rename-desktop-file: org.qt-project.qtdesignstudio.desktop
```

Comment:
I don't see why you would not use this app-id.

Reply:
Seems resolved without any answer. If the app internally uses that id, the flatpak should use the same.

Reply:
Because flatpak bilder is complaining about the '-'.

Reply:
You demangle it dashes should be converted to underscore.

Reply:
Yes, but flatpak builder was not working.

Reply:
error: 'org.qt-project.qtdesignstudio' is not a valid application name: Only last name segment can contain -

Reply:
That's what I meant it should've been: `org.qt_project.qtdesignstudio`

Reply:
But that is not the domain name, so I chose the other domain name that is under our control.

Reply:
Dashes in the id are to be converted to underscore that's the rule for application ids. When converting them back to domain name, implementations reverse it.

See the appstream specification for id tag.

Reply:
Oh, I love it. ;-)

So I abort this request and start a new one?

Reply:
I changed the app id upstream.